lib/wallet/src/Cardano/Wallet.hs

```diff
@@ -2982,11 +3032,32 @@ calculateFeePercentiles
         ErrSelectAssetsSelectionError
             ( SelectionBalanceErrorOf
                 ( UnableToConstructChange
-                    ( UnableToConstructChangeError {requiredCost})
+                    UnableToConstructChangeError{requiredCost}
                 )
             ) -> pure $ Fee requiredCost
         e -> throwE e
 
+-- | Make a pair of fee estimation percentiles more imprecise.
```

Comment:
Suggestion: explain more precisely how this function makes the percentiles less precise.

```suggestion
-- | Make a pair of fee estimation percentiles more imprecise.
--
-- For a given number of bytes `n` and interval `(p, q)`, this function
-- computes the superinterval `(p - x, q + x)`, where `x` is the cost of
-- encoding `n` bytes according to the given protocol parameters.
```